sim/src/main/cc/firesim/firesim_top.cc

```diff
@@ -568,9 +568,6 @@ void firesim_top_t::run() {
     fprintf(stderr, "Commencing simulation.\n");
     record_start_times();
 
-    // Assert reset T=0 -> 50
```

Comment:
Why was this removed?

Reply:
The resetPulseBridge now drives the target reset -- this resolves the reset deadlock bug you saw at one point, because this function can no-longer prevent execution of the simulator's main loop.